ee/backend/app/build.gradle

```diff
@@ -99,6 +99,11 @@ dependencies {
      */
     implementation 'org.apache.poi:poi-ooxml:5.3.0'
     implementation 'org.apache.commons:commons-collections4:4.4'
+
+    /**
+     * HANDLEBARS
+     */
+    implementation('com.github.jknack:handlebars:4.4.0')
```

Comment:
Still there...

Reply:
> Can we keep the old version in root then?

New version is required for to work with updated Kotlin version

> Still there...

Ah, didn't notice it was in two places (?)